frontend/src/models/DeliveryModel.tsx

```diff
@@ -1,4 +1,8 @@
 export type DeliveryModel = {
     id: string,
     title: string
+}
+
+export type NewDeliveryModel = {
+    title: string
```

Comment:
Man könnte hier darüber nachdenken, ob man nicht aus diesen beiden models eins macht und die id als optionalen Parameter angibt, was aber auch so durchaus in Ordnung ist